jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy

```diff
@@ -319,6 +325,11 @@ def run_make_coverage() {
             unstash 'integration-cov-shardddl3'
             unstash 'integration-cov-shardddl4'
             unstash 'integration-cov-sharding sequence_sharding'
+            unstash 'integration-cov-new_relay'
+            unstash 'integration-cov-import_v10x'
```

Comment:
If these cases will be called here, they should be moved out of `others_integration.txt` in DM's code repo.

Please let us review any changes before you actually change CI. Don't let disable release-2.0's CI happen again.

Reply:
I know this, this pr will not be merged before this [one](https://github.com/pingcap/dm/pull/1728)